config/examples/rolling_updates.yml

```diff
@@ -6,7 +6,6 @@ base: &default
   duplicate_application_id: true
   duplicate_user_key: true
   instant_bill_plan_change: false
-  new_notification_system: true
```

Comment:
Do we need some special update steps for saas or on-prem about this?

Also shouldn't we remove `NewNotificationSystem` from  `app/lib/logic/rolling_updates.rb`?

Reply:
Thanks, didn't see this removal.

I have some recollection that if the configuration is not removed from config file, and the class is missing, then 3scale will not start. That's also what I meant, do we need special upgrade instructions for example?

Reply:
Well, I tried adding the config back in this branch, where the class doesn't exist anymore, and everything seems to work fine. We can add a release note to heads up the clients just in case.